app/gui/view/graph-editor/src/component/node/input/area.rs

```diff
@@ -286,6 +286,15 @@ impl Model {
         self
     }
 
+    /// Return a list of Node's input ports.
+    pub fn ports(&self) -> Vec<port::Model> {
```

Comment:
Why `input_port` returns `input::port::Shape` while `ports` return Vector of `port::Model`? This is inconsistent.

Reply:
I renamed `input_port` to `input_port_shape`